tinkerpop3/src/main/java/io/shiftleft/overflowdb/storage/OdbStorage.java

```diff
@@ -98,13 +101,70 @@ public NodeSerializer getNodeSerializer() {
 
   public MVMap<Long, byte[]> getNodesMVMap() {
     if (mvstore == null) {
-      mvstore = new MVStore.Builder().fileName(mvstoreFile.getAbsolutePath()).open();
-      nodesMVMap = mvstore.openMap("nodes");
+      mvstore = initializeMVStore();
     }
+    if (nodesMVMap == null)
+      nodesMVMap = mvstore.openMap("nodes");
     return nodesMVMap;
   }
 
+  private MVStore initializeMVStore() {
+    final MVStore store = new MVStore.Builder().fileName(mvstoreFile.getAbsolutePath()).open();
+    return store;
+  }
+
+  private Map<String, String> getIndexNameMap(MVStore store) {
+    store.getMapNames().stream().forEach(name -> System.out.println("name: " + name));
+    return store
+        .getMapNames()
+        .stream()
+        .filter(s -> s.startsWith(INDEX_PREFIX))
+        .map(s -> {System.out.println("filtered name: " + s); return s;})
+        .collect(Collectors.toConcurrentMap(s -> removeIndexPrefix(s), s -> s));
+  }
+
+  public Set<String> getIndexNames() {
+    return getIndexNameMap(mvstore).keySet();
+  }
+
+  private String removeIndexPrefix(String s) {
+    assert s.startsWith(INDEX_PREFIX);
```

Comment:
same remark about `assert` as above applies

Reply:
Yes, I'll remove all asserts, they are just a leftover after I was debugging it :)